Mage.Sets/src/mage/cards/l/LethalScheme.java

```diff
@@ -0,0 +1,176 @@
+package mage.cards.l;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import javafx.util.Pair;
```

Comment:
Please re-write without the Javafx dependency. See e.g. for suggestions on alternatives to Pair https://stackoverflow.com/questions/61516130/why-is-there-no-javafx-util-pair-is-this-a-bug-in-eclipse